src/flash/com/longtailvideo/jwplayer/controller/PluginLoader.as

```diff
@@ -27,17 +27,19 @@ import flash.utils.Dictionary;
  */
 public class PluginLoader extends EventDispatcher {
 
+    public var plugins:Object;
+
+    private var _remainingLoaders:uint;
+    private var _errorState:Boolean;
+
     public function PluginLoader() {
-        loaders = new Dictionary();
+        _remainingLoaders = 0;
```

Comment:
It's technically the same, but imho it's tidier. One fewer line and the definition includes the initial state.